matlab/MeCorr/MATLAB_CODE_allrxns.m

```diff
@@ -0,0 +1,419 @@
+      %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

```

Comment:
Rename the three files with `MATLAB_CODE_*` files - it's not clear from the get-go what the difference between allrxns, me, and methyl are without opening and carefully reading the file contents.